go.mod

```diff
@@ -27,7 +27,7 @@ require (
 	github.com/pingcap/log v0.0.0-20210906054005-afc726e70354
 	github.com/pingcap/parser v0.0.0-20210907051057-948434fa20e4
 	github.com/pingcap/tidb v1.1.0-beta.0.20210914112841-6ebfe8aa4257
-	github.com/pingcap/tidb-tools v5.2.0-alpha.0.20210727084616-915b22e4d42c+incompatible
+	github.com/pingcap/tidb-tools v5.2.1+incompatible
```

Comment:
why update dependency in here? this may cause conflicts with other PR 🤣
please remove this part, if you want to upgrade dep, you can do it in another PR

Reply:
It's automaticlly changed by vscode, and I forgot to check this.
Thanks for your advice, I will remove this part in next commit.